src/main/java/com/coolkids/coolKidsApp/controllers/EventController.java

```diff
@@ -11,31 +13,16 @@
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
 
+@RestController
 @RequestMapping(path = "api/v1/")
 @AllArgsConstructor
 public class EventController {
+    private final EventService eventService;
 
-    //Todo: list all events
-
-
-
-    //Todo: create event
-
-
-    //Todo: delete event
-
-
-    //Todo: get an event by id
-
-
-    //Todo: update an event (patch)
-
-
-    //Todo: cancel an event
-
-
-    //Todo: get users signed up for an event
-
-
+    @PostMapping("/events") //Getmapping vs Postmapping here?
+    public ResponseEntity<EventListDTO> getAllEvents(){
+        return new ResponseEntity<EventListDTO>(
+                new EventListDTO(eventService.getAllEvents()), HttpStatus.OK);
+    }
```

Comment:
can you please undo the deletions of the todos? We could use those to track our progress on the services we need.